client/.coderabbit.yaml

```
@@ -20,6 +20,19 @@ review:
     - name: 불필요한 리렌더링 감지
       enabled: true
   summary: true
+  path_instructions:
+    - path: '**/*'
+      instructions: |
+        모든 리뷰 코멘트와 피드백을 한국어로 작성해주세요. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)